client/my-sites/stats/activity-log-item/activity-title.jsx

```diff
@@ -0,0 +1,187 @@
+import { localize } from 'i18n-calypso';
+
+class ActivityTitle extends Component {
+
+	static propTypes = {
+		group: PropTypes.oneOf( [
+			'attachment',
+			'comment',
+			'core',
+			'plugin',
+			'post',
+			'term',
+			'theme',
+			'user',
+			'widget',
+		] ).isRequired,
+		name: PropTypes.string.isRequired,
+
+		actor: PropTypes.shape( {
+			display_name: PropTypes.string,
+			login: PropTypes.string,
+		} ),
+
+		object: PropTypes.shape( {
+			attachment: PropTypes.shape( {
+				mime_type: PropTypes.string,
+				id: PropTypes.number.isRequired,
+				title: PropTypes.string.isRequired,
+				url: PropTypes.shape( {
+					host: PropTypes.string.isRequired,
+					url: PropTypes.string.isRequired,
+					host_reversed: PropTypes.string.isRequired,
+				} ).isRequired,
+			} ),
+
+			comment: PropTypes.shape( {
+				approved: PropTypes.bool.isRequired,
+				id: PropTypes.number.isRequired,
+			} ),
+
+			core: PropTypes.shape( {
+				new_version: PropTypes.string,
+				old_version: PropTypes.string,
+			} ),
+
+			plugin: PropTypes.oneOfType( [
+				PropTypes.shape( {
+					name: PropTypes.string,
+					previous_version: PropTypes.string,
+					slug: PropTypes.string,
+					version: PropTypes.string,
+				} ),
+				PropTypes.arrayOf(
+					PropTypes.shape( {
+						name: PropTypes.string,
+						previous_version: PropTypes.string,
+						slug: PropTypes.string,
+						version: PropTypes.string,
+					} ),
+				),
+			] ),
+
+			post: PropTypes.shape( {
+				id: PropTypes.number.isRequired,
+				status: PropTypes.string.isRequired,
+				type: PropTypes.string,
+				title: PropTypes.string,
+			} ),
+
+			term: PropTypes.shape( {
+				id: PropTypes.number.isRequired,
+				title: PropTypes.string.isRequired,
+				type: PropTypes.string.isRequired,
+			} ),
+
+			theme: PropTypes.oneOfType( [
+				PropTypes.arrayOf(
+					PropTypes.shape( {
+						name: PropTypes.string,
+						slug: PropTypes.string,
+						uri: PropTypes.string,
+						version: PropTypes.string,
+					} )
+				),
+				PropTypes.shape( {
+					name: PropTypes.string,
+					slug: PropTypes.string,
+					uri: PropTypes.string,
+					version: PropTypes.string,
+				} ),
+			] ),
+
+			user: PropTypes.shape( {
+				display_name: PropTypes.string,
+				external_user_id: PropTypes.string,
+				login: PropTypes.string,
+				wpcom_user_id: PropTypes.number,
+			} ),
+
+			widget: PropTypes.shape( {
+				id: PropTypes.number,
+				name: PropTypes.string,
+				sidebar: PropTypes.string,
+			} ),
+		} ).isRequired,
+
+		// localize
+		moment: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	getActorName() {
+		const displayName = get( this.props, [ 'actor', 'display_name' ] );
+		if ( displayName ) {
+			return displayName;
+		}
+
+		const login = get( this.props, [ 'actor', 'login' ] );
+		if ( login ) {
+			return login;
+		}
+	}
+
+	getPostName() {
+		return get( this.props, [ 'object', 'post', 'title' ], this.props.translate( 'A post' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'a post.', { context: 'Someone commented on a post.'} )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).